app/pb_kits/playbook/pb_button/_button.jsx

```diff
@@ -1,8 +1,9 @@
 /* @flow */
 
-import React, { Component } from 'react'
```

Comment:
Is playbook defaulting to single quotes @thestephenmarshall ?

Reply:
Yes